src/components/CustomerQuota/CustomerQuotaScreen.tsx

```diff
@@ -195,7 +195,7 @@ export const CustomerQuotaScreen: FunctionComponent<CustomerQuotaProps> = ({
       return;
     }
 
-    if (govWalletBalanceError) {
+    if (quotaState === "DEFAULT" && govWalletBalanceError) {
```

Comment:
We only check for GovWallet balance related errors if there are no quota errors.